ui/src/components/view/SearchView.vue

```diff
@@ -347,6 +348,14 @@ export default {
         this.fillFormFieldValues()
       })
     },
+    sortArray (data) {
```

Comment:
Could you pass the sort key to the method? That way the domains can be sorted by path and the rest by name ?